rero_ils/modules/documents/serializers/ris.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2022 RERO
+# Copyright (C) 2022 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""RIS Document serialization."""
+
+from flask import current_app, request, stream_with_context
+from invenio_i18n.ext import current_i18n
+
+from rero_ils.modules.documents.api import Document
+from rero_ils.modules.documents.serializers.base import \
+    BaseDocumentFormatterMixin
+from rero_ils.modules.documents.utils import create_contributions
+from rero_ils.utils import get_i18n_supported_languages
+
+
+class RISSerializer:
+    """BibTeX serializer for records."""
+
+    def serialize(self, pid, record, links_factory=None, **kwargs):
+        """Serialize a single record and persistent identifier.
+
+        :param pid: Persistent identifier instance.
+        :param record: Record instance.
+        :param links_factory: Factory function for record links.
+        """
+        Document.post_process(record)
+        record = record.replace_refs()
+        contributions = create_contributions(
+            record.get('contribution', [])
+        )
+        if contributions:
+            record['contribution'] = contributions
+        return RISFormatter(record=record).format()
```

Comment:
I recently learned that you can also write it like this:
```python
if contributions := create_contributions(record.get('contribution', [])):
    record['contribution'] = contributions
```

Reply:
done. Thanks!